app/preview/ui.test.js

```diff
@@ -130,14 +130,19 @@ test('when existing marketing data - The client application type section and bro
-    .expect(supportedBrowsersRow.find('.nhsuk-summary-list__key').innerText).eql('Browsers Supported')
-    .expect(supportedBrowsersRow.find('.nhsuk-summary-list__value').innerText).eql('Google Chrome\nMozilla Firefox')
-    .expect(mobileResponsiveRow.find('.nhsuk-summary-list__key').innerText).eql('Mobile responsive')
-    .expect(mobileResponsiveRow.find('.nhsuk-summary-list__value').innerText).eql('Yes')
-    .expect(pluginsRequiredRow.find('.nhsuk-summary-list__key').innerText).eql('Plug-ins or extensions required')
-    .expect(pluginsRequiredRow.find('.nhsuk-summary-list__value').innerText).eql('Yes')
-    .expect(pluginsDetailRow.find('.nhsuk-summary-list__key').innerText).eql('Plug-ins or extensions information')
-    .expect(pluginsDetailRow.find('.nhsuk-summary-list__value').innerText).eql('The plugin detail')
-    .expect(hardwareRequirementsDescriptionRow.find('.nhsuk-summary-list__key').innerText).eql('Hardware requirements')
-    .expect(hardwareRequirementsDescriptionRow.find('.nhsuk-summary-list__value').innerText).eql('Some hardware requirement description');
+    .expect(supportedBrowsersRow.find('div[data-test-id="view-section-table-row-title"]').innerText).eql('Browsers Supported')
+    .expect(supportedBrowsersRow.find('div[data-test-id="view-section-table-row-component"]').innerText).eql('Google Chrome\nMozilla Firefox')
+    .expect(supportedBrowsersRow.find('div[data-test-id="view-section-table-row-horizontal"]').exists).ok()
+    .expect(mobileResponsiveRow.find('div[data-test-id="view-section-table-row-title"]').innerText).eql('Mobile responsive')
+    .expect(mobileResponsiveRow.find('div[data-test-id="view-section-table-row-component"]').innerText).eql('Yes')
+    .expect(mobileResponsiveRow.find('div[data-test-id="view-section-table-row-horizontal"]').exists).ok()
+    .expect(pluginsRequiredRow.find('div[data-test-id="view-section-table-row-title"]').innerText).eql('Plug-ins or extensions required')
+    .expect(pluginsRequiredRow.find('div[data-test-id="view-section-table-row-component"]').innerText).eql('Yes')
+    .expect(pluginsRequiredRow.find('div[data-test-id="view-section-table-row-horizontal"]').exists).ok()
+    .expect(pluginsDetailRow.find('div[data-test-id="view-section-table-row-title"]').innerText).eql('Plug-ins or extensions information')
+    .expect(pluginsDetailRow.find('div[data-test-id="view-section-table-row-component"]').innerText).eql('The plugin detail')
+    .expect(pluginsDetailRow.find('div[data-test-id="view-section-table-row-horizontal"]').exists).ok()
+    .expect(hardwareRequirementsDescriptionRow.find('div[data-test-id="view-section-table-row-title"]').innerText).eql('Hardware requirements')
+    .expect(hardwareRequirementsDescriptionRow.find('div[data-test-id="view-section-table-row-component"]').innerText).eql('Some hardware requirement description')
+    .expect(hardwareRequirementsDescriptionRow.find('div[data-test-id="view-section-table-row-vertical"]').exists).ok();
```

Comment:
Nice! Do you think it's worth adding a blank line between each row to improve the readability of this